setup.sh

```diff
@@ -177,13 +183,18 @@ installDL() {
     read installDL trash
     case $installDL in
       [yY])
+        if [ $EUID != 0 ]; then
+          write "[ℹ] Installing cUrl needs root privileges."
+          write "    Since you are not running this script with root privileges"
+          write "    you are likely to be prompted for your sudo password."
+        fi
         echo
         if commandExists apt-get; then
-          apt-get -y install curl
+          sudo apt-get -y install curl
```

Comment:
Everywhere `sudo` is used:

1. `sudo` may not be installed on the system. If the script is run as root, we can just skip it
2. if the script is run as a standard user, I would display him all sudo commands and ask for confirmation (Enter is good enough is this case) before proceeding.

Reply:
we should discuss about this